etcdctlv3/command/make_mirror_command.go

```diff
@@ -60,7 +60,9 @@ func makeMirrorCommandFunc(cmd *cobra.Command, args []string) {
 	dc := mustClient([]string{args[0]}, mmcert, mmkey, mmcacert)
 	c := mustClientFromCmd(cmd)
 
-	err := makeMirror(context.TODO(), c, dc)
+	ctx, cancel := commandCtx(cmd)
```

Comment:
make mirror is also a long running command. we should not set the timeout for it.